torchvision/models/_utils.py

```diff
@@ -1,6 +1,7 @@
 from collections import OrderedDict
 
 import torch
+from torch import Tensor
```

Comment:
Given that we don't need to use this in so many places and we already importing `torch`, couldn't we simply use `torch.Tensor`?